viewer/packages/360-images/src/Image360Facade.ts

```diff
@@ -25,7 +25,11 @@ export class Image360Facade<T> {
     this._image360Collections.forEach(collection => collection.setIconsVisibility(visible));
   }
 
-  set allHoverIconsVisibility(visible: boolean) {
+  set allIconsSelected(visible: boolean) {
+    this._image360Collections.forEach(collection => collection.setSelectedForAll(visible));
+  }
+
+  set hoverIconVisibility(visible: boolean) {
     this._image360Collections.forEach(collection => collection.setSelectedVisibility(visible));
   }
```

Comment:
Might need an explanation for this too :monocle_face:

Reply:
`allIconsSelected` refers to setting `selected` property of an icon. `hoverIconVisibility` sets visibility of hover icon sprite since with new implementation there is only one sprite that "moves" around the icons when you hover over them. A little non-obvious naming inside `hoverIconVisibility` comes from the fact that `setSelectedVisibility` is a public method, so we can't break the api in this PR.